gclient/src/node/port.rs

```diff
@@ -0,0 +1,19 @@
+//! Picking random ports
+use rand::Rng;
+use std::{net::TcpListener, ops::Range};
+
+/// localhost addr
+const LOCALHOST: &str = "127.0.0.1";
+const PORT_RANGE: Range<u16> = 15000..25000;
+
+/// Pick a random port
+pub fn pick() -> u16 {
```

Comment:
Can we use port 0 for dynamic ports instead of RNG? Or does it not work with the node?